src/Microsoft.CodeAnalysis.Analyzers/CSharp/MetaAnalyzers/Fixers/CSharpDoNotCompareSyntaxTokenAgainstDefaultLiteralFix.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.Fixers;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.CSharp.Analyzers.MetaAnalyzers.Fixers
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(CSharpApplyDiagnosticAnalyzerAttributeFix)), Shared]
+    internal sealed class CSharpDoNotCompareSyntaxTokenAgainstDefaultLiteralFix : DoNotCompareSyntaxTokenAgainstDefaultLiteralFix
+    {
+        protected override ISyntaxFacts SyntaxFacts => CSharpSyntaxFacts.Instance;
```

Comment:
This doesn't compile currently as this layer doesn't have access to CSharpSyntaxFacts. Is it okay if I add a reference to `Refactoring.CSharp.Utilities` which contains `CSharpSyntaxFacts`?

I also did a similar thing to access `ISyntaxFacts`, was that okay too?